assets/queries/terraform/alicloud/slb_policy_with_insecure_tls_version_in_use/metadata.json

```
@@ -0,0 +1,11 @@
+{
+  "id": "dbfc834a-56e5-4750-b5da-73fda8e73f70",
+  "queryName": "SLB Policy With Insecure Tls Version In Use",
+  "severity": "MEDIUM",
+  "category": "Encryption",
+  "descriptionText": "SLB Policy should not support insecure versions of Tls protocol",
```

```suggestion
  "queryName": "SLB Policy With Insecure TLS Version In Use",
  "severity": "MEDIUM",
  "category": "Encryption",
  "descriptionText": "SLB Policy should not support insecure versions of TLS protocol",
```

for standard purposes